docs/http-api.md

```diff
@@ -163,6 +163,26 @@ POST /metrics/delete
 curl -H "X-Org-Id: 12345" --data query=statsd.fakesite.counters.session_start.*.count "http://localhost:6060/metrics/delete"
 ```
 
+
+## Deleting tagged metrics
+
+This will delete the metrics (technically metricdefinitions) matching the `path` parameter(s).
+Note that the data stays in the datastore until it expires.
+Should the metrics enter the system again with the same metadata, the data will show up again.
+
+```
+POST /tags/delSeries
+```
+
+* header `X-Org-Id` required
+* path (required, multiple allowed): A single Graphite series
+
+#### Example
+
+```bash
+curl -H "X-Org-Id: 12345" -d path=some.series;key=value -d <span class="x x-first x-last">path=</span>another.series;tag=value "http://localhost:6060/tags/delSeries"
```

Comment:
what's the html tag in this line? i assume that's an accident?

Reply:
No idea, I think that might have been intellij trying to be helpful?
1a0bcfcf44ffce96c5bd1f774cc748459aceefe4